src/containers/Masthead/components/MastheadTopics.jsx

```diff
@@ -36,7 +36,16 @@ const MastheadTopics = props => {
     getSelectedTopic(expandedTopicIds),
     topicResourcesByType,
     activeFilters.join(','),
-  );
+  ).sort((topicA, topicB) => {
```

Comment:
Sort on name will give wrong order. There is a rank field on some request.

Reply:
Should be added in a separate PR.

Reply:
Ah, okey, I will remove it.